src/pybrake/metrics.py

```diff
@@ -1,16 +1,15 @@
-import json
-import urllib.request
-import urllib.error
 import threading
 from contextlib import contextmanager
 import time as pytime
 
-from . import constant
 from .utils import logger
 
 
 threadLocal = threading.local()
 
+FLUSH_PERIOD = 15
+_METRIC_KEY = "_ab_metric"
```

Comment:
I don't understand what's going on. A couple commits ago we moved this to `constants` and now we are moving it back?